presto-parser/src/main/java/com/facebook/presto/sql/tree/RenameTable.java

```diff
@@ -42,6 +42,12 @@ private RenameTable(Optional<NodeLocation> location, QualifiedName source, Quali
         this.target = requireNonNull(target, "target name is null");
     }
 
+    @Override
+    public QualifiedName getQualifiedName()
```

Comment:
Any thoughts how to unify `getQualifiedName` without introducing such a misleading code?
I left `getSource` to make it clear though.
`getQualifiedName` is just for `beginQuery` and in only that context is used.